content/applications/websites/website/pages/menus.rst

```diff
@@ -0,0 +1,88 @@
+=====
+Menus
+=====
+
+Menus are used to organize your website’s content and help visitors navigate through your web pages
+effectively. User-friendly and well-structured website menus also play a crucial role in improving
+:doc:`search engine rankings <seo>`.
+
+Odoo allows you to customize the content and appearance of your website's menu to your needs.
+
+Menu editor
+===========
+
+The menu editor allows you to edit your website's menu and add regular menu items and mega menus.
+
+To edit your website's menu, go to :menuselection:`Website --> Site --> Menu Editor`. From there,
+you can:
+
+- **rename** a menu item or change its URL using the :guilabel:`Edit Menu Item` icon;
+- **delete** a menu item using the :guilabel:`Delete Menu Item` icon;
+- **move** a menu item by dragging and dropping it to the desired place in the menu;
+- **create a regular drop-down menu** by dragging and dropping the sub-menu items to the right,
+  underneath their parent menu.
+
+.. image:: menus/menu-editor.png
+   :alt: Menu editor with sub-menus
+
+.. note::
+
+  You can also access the menu editor by clicking :guilabel:`Edit`, selecting any menu item and
+  clicking the :guilabel:`Edit menu` icon.
+
+  .. image:: menus/edit-menu-icon.png
+     :alt: Access the Menu editor while in Edit mode.
+
+Adding regular menu items
+-------------------------
+
+By default, pages are added to the menu as regular menu items when :doc:`they are created
+<../pages>`. You can also add regular menu items from the menu editor by clicking :guilabel:`Add
+Menu Item`. Enter the :guilabel:`Name` and URL of the related page in the pop-up window that appears
+on the screen and click :guilabel:`OK`.
+
+.. tip::
+   In the :guilabel:`URL or Email` field, you can type `/` to search for a page on your website or
+   `#` to search for an existing custom anchor.
+
+Adding mega menus
```

Comment:
Maybe just `Mega menus` since it's about adding and editing their appearance in the website builder?

Or change the structure, for example:

```restructuredtext
=====
Menus
=====

Menu editor
===========

Regular menu
============

Mega menu
=========

Header and navigation bar
=========================
```

Reply:
I think I want to keep it as such as 1) it's still about *adding* menu items (for both regular and mega menus), 2) you can't really add a mega menu without customizing it (e.g. changing the URL and name of the options) - well, you can but IMO that would be useless :) - and 3) to me, both belong in the menu editor part as it is the "starting point" for both. Does that make sense to you?